client/rpc/src/system/mod.rs

```diff
@@ -98,7 +98,7 @@ impl<B: traits::Block> System<B> {
 
 		// Get the chain's type.
 		rpc_module
-			.register_method("system_ChainType", |_, system| Ok(system.info.chain_type.clone()))?;
+			.register_method("system_chainType", |_, system| Ok(system.info.chain_type.clone()))?;
```

Comment:
oops!

I wonder if we should validate the method names in `jsonrpsee`. Ideally at compile time, but a runtime error would work too.

Reply:
I rather not

> method
    A String containing the name of the method to be invoked. Method names that begin with the word rpc followed by a period character (U+002E or ASCII 46) are reserved for rpc-internal methods and extensions and MUST NOT be used for anything else. 
 
Unless we could have some attribute on the `proc macro` with naming convention such as `camelCase` or whatever :P

Reply:
Right, it could be a future extension. We could perhaps at least enforce method names contain no whitespace?